### ISSUE TITLE
Crash when requesting OAUTH2 callback URL directly with curl

### ISSUE THREAD

**Issue by jeffmahoney** [Velocidex/velociraptor#1999]
`$ curl https://vrrserver/auth/github/callback`

leads to:

```
 [INFO] 2022-08-19T01:45:30Z http: panic serving 172.31.0.4:38558: runtime error: invalid memory address or nil pointer dereference
 goroutine 412 [running]:
 net/http.(*conn).serve.func1()
 	/usr/lib64/go/1.18/src/net/http/server.go:1825 +0xbf
 panic({0x1bd9580, 0x3073280})
 	/usr/lib64/go/1.18/src/runtime/panic.go:844 +0x258
 www.velocidex.com/golang/velociraptor/api/authenticators.(*GitHubAuthenticator).oauthGithubCallback.func1({0x22f3a88, 0xc0006fa8c0}, 0xc00087ad00)
 	/home/abuild/rpmbuild/BUILD/velociraptor-0.6.4~git31.4298eab0/api/authenticators/github.go:108 +0x94
 net/http.HandlerFunc.ServeHTTP(0x7f9a02a24088?, {0x22f3a88?, 0xc0006fa8c0?}, 0x42b5e5?)
 	/usr/lib64/go/1.18/src/net/http/server.go:2084 +0x2f
 net/http.(*ServeMux).ServeHTTP(0x0?, {0x22f3a88, 0xc0006fa8c0}, 0xc00087ad00)
 	/usr/lib64/go/1.18/src/net/http/server.go:2462 +0x149
 net/http.serverHandler.ServeHTTP({0xc0026f3a10?}, {0x22f3a88, 0xc0006fa8c0}, 0xc00087ad00)
 	/usr/lib64/go/1.18/src/net/http/server.go:2916 +0x43b
 net/http.(*conn).serve(0xc002595cc0, {0x22f4fc0, 0xc002561c80})
 	/usr/lib64/go/1.18/src/net/http/server.go:1966 +0x5d7
 created by net/http.(*Server).Serve
 	/usr/lib64/go/1.18/src/net/http/server.go:3071 +0x4db 

```

**Comment by scudette** [comment]
Very good catch! Thanks!

**Comment by scudette** [comment]
I pushed the fix to the 0.6.5 branch too and  released a 0.6.5-3 build.

Thanks for reporting this! Great catch.
